src/main/resources/public/README.md

```diff
@@ -16,3 +16,24 @@ npm run build
 ```
 
 For detailed explanation on how things work, consult the [docs for vue-loader](http://vuejs.github.io/vue-loader).
+
+### Build Setup Yarn
+``` bash
+# install dependencies
+yarn install
+
+# serve with hot reload at localhost:8080
+yarn dev
+
+# build for production with minification
+yarn build
+```
+
+#### Added dependancies
```

Comment:
Good idea.

However, it could be placed in a Changelog.md for easier followup (and not over polluting the README file)